ReleaseTooling/ZipBuilderV2Plan.md

```diff
@@ -1,17 +1,18 @@
-V2 will add an option to generate a zip distribution of binary frameworks from an arbitrary list
```

Comment:
Maybe we should keep V2 and start V3 instead?

Reply:
Sure, I can do that. Wasn't sure if it's relevant to still have things in the "plan" doc that were built already.